pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -108,11 +110,12 @@ func NewMustGatherCommand(f kcmdutil.Factory, streams genericclioptions.IOStream
 
 func NewMustGatherOptions(streams genericclioptions.IOStreams) *MustGatherOptions {
 	return &MustGatherOptions{
-		SourceDir: "/must-gather/",
-		IOStreams: streams,
-		LogOut:    newPrefixWriter(streams.Out, "[must-gather      ] OUT"),
-		RawOut:    streams.Out,
-		Timeout:   10 * time.Minute,
+		SourceDir:  "/must-gather/",
+		IOStreams:  streams,
+		LogOut:     newPrefixWriter(streams.Out, "[must-gather      ] OUT"),
+		RawOut:     streams.Out,
+		Timeout:    10 * time.Minute,
+		VolumeSize: "3Gi",
```

Comment:
@tssurya @sferich888 you did a good amount of analysis about the must-gather pod's volume size usage on this https://github.com/openshift/must-gather/pull/370. Would you mind, if I ask your opinions about this default `3Gi`?.

Reply:
If we collect more than 3 GB I think we have a problem; In general we collect 800 MB to 1.5 GB (if I recall). So a 3 GB limit gives us some buffer (however I don't think we really should be collecting that much data); we really need to look at how to make this smaller (but that is for a different ticket).